.github/scripts/get_version.py

```
@@ -0,0 +1,41 @@
+import sys
+import xml.etree.ElementTree as ET
+
+XML_FILE_PATH = "sherlock-branding/resources/idea/SherlockPlatformApplicationInfo.xml"
+NAMESPACE_URI = "http://jetbrains.org/intellij/schema/application-info"
+
+
+def get_version_from_xml() -> str:
+  """
+  Reads the SherlockPlatformApplicationInfo.xml file, extracts the version,
+  and returns it in 'v1.0.0' format.
+  """
+  try:
+    tree = ET.parse(XML_FILE_PATH)
+    root = tree.getroot()
+    version_element = root.find(f'{{{NAMESPACE_URI}}}version')
+    if version_element is not None:
+      major = version_element.get("major")
+      minor = version_element.get("minor")
+      patch = version_element.get("patch", default='0')
+      if major is not None and minor is not None:
+        return f"v{major}.{minor}.{patch}"
+
+  except ET.ParseError as e:
+    print(f"Error parsing XML file: {e}", file=sys.stderr)
+
```

I believe these except cases should re-raise the exception so that `main` will catch and exit with exit_code=1.

i.e, add `raise` here